### PR TITLE
ios: update podfile for CocoaPods 1.6.0

### PR DESCRIPTION
**Pull request by saghul**

### CODE REVIEW

File: ios/sdk/sdk.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -63,7 +63,7 @@
 		0BB9AD781F5EC6D7001C08DB /* Intents.framework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.framework; name = Intents.framework; path = System/Library/Frameworks/Intents.framework; sourceTree = SDKROOT; };
 		0BB9AD7A1F5EC8F4001C08DB /* CallKit.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = CallKit.m; sourceTree = "<group>"; };
 		0BB9AD7C1F60356D001C08DB /* AppInfo.m */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.c.objc; path = AppInfo.m; sourceTree = "<group>"; };
-		0BC4B8681F8C01E100CE8B21 /* CallKitIcon.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; name = CallKitIcon.png; path = ../../react/features/mobile/call-integration/CallKitIcon.png; sourceTree = "<group>"; };
+		0BC4B8681F8C01E100CE8B21 /* CallKitIcon.png */ = {isa = PBXFileReference; lastKnownFileType = image.png; name = CallKitIcon.png; path = "../../react/features/mobile/call-integration/CallKitIcon.png"; sourceTree = "<group>"; };
```
Comment (zbettenbuk) [inline ios/sdk/sdk.xcodeproj/project.pbxproj]: I guess is wasn't you directly, but is this change necessary? We don't seem to have any paths quoted (see below ../../sounds/joined.wav and similar paths).

Comment (saghul) [reply ios/sdk/sdk.xcodeproj/project.pbxproj]: Necessary? No. But Xcode decided to make it after I updated CocoaPods and I don't want to be playing pingpong with it :-)

**Comment by saghul** [comment]
Updated with some other unneeded-but-Xcode-wants-them minor changes, PTAL.

**Comment by saghul** [comment]
Thanz bez!
